libraries/classes/InsertEdit.php

```diff
@@ -693,6 +693,7 @@ private function getHtmlInput(
             }
         }
 
+        $inputMode = '';
         $inputMinMax = '';
         if (in_array($column['True_Type'], $this->dbi->types->getIntegerTypes())) {
```

Comment:
You can assign the condition value to use it later to add the input mode. 

```suggestion
        $isInteger = in_array($column['True_Type'], $this->dbi->types->getIntegerTypes());
        if ($isInteger) {
```